newIDE/app/src/MainFrame/index.js

```diff
@@ -2343,6 +2343,8 @@ const MainFrame = (props: Props) => {
         return saveProjectAs();
       }
 
+      if (!!currentFileMetadata.ownerId) return;
```

Comment:
for the sake of clarity, maybe name this `isProjectOwnedBySomeoneElse = !!currentFileMetadata.ownerId`

Reply:
Good idea!